services/data_plane_manager/src/main/java/com/futurewei/alcor/dataplane/service/impl/OVSGoalStateServiceImpl.java

```diff
@@ -0,0 +1,103 @@
+/*
+Copyright 2019 The Alcor Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+        you may not use this file except in compliance with the License.
+        You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+        Unless required by applicable law or agreed to in writing, software
+        distributed under the License is distributed on an "AS IS" BASIS,
+        WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+        See the License for the specific language governing permissions and
+        limitations under the License.
+*/
+package com.futurewei.alcor.dataplane.service.impl;
+
+import com.futurewei.alcor.common.message.MessageClient;
+import com.futurewei.alcor.dataplane.config.Config;
+import com.futurewei.alcor.dataplane.config.grpc.GoalStateProvisionerClient;
+import com.futurewei.alcor.dataplane.service.GoalStateService;
+import com.futurewei.alcor.schema.Goalstate;
+import com.futurewei.alcor.schema.Goalstateprovisioner;
+
+public class OVSGoalStateServiceImpl implements GoalStateService {
+    public static int GRPC_SERVER_PORT = 50001;
```

Comment:
Let us have GRPC_SERVER_PORT as a configuration in application.properties so that users could configure it. 

@er1cthe0ne this is the default gPRC server port value that ACA needs to expose. Same as before.

Reply:
would move to properties file